cmd/system-probe/modules/network_tracer_nolinux_bpf.go

```diff
@@ -0,0 +1,15 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
maybe this file should be `network_tracer_no_linux_bpf` or `network_tracer_linux_bpf_unsupported`?